docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -143,6 +143,48 @@ To send a command from the client and to process it on the server side, follow t
     }
     ```
 
+To send a search command from the client and to process it on the server side, follow these steps:
+1.  To send a search command on the client side, use the `RequestAsync()` method of the [Tizen.Multimedia.Remoting.MediaController](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaController.html) class:
+
+    ```csharp
+    var searchCondition = new MediaControlSearchCondition(MediaControlContentType.Image,
+        MediaControlSearchCategory.Artist, "GD", null);
+
+    mediaController.RequestAsync(new SearchCommand(searchCondition));
+    ```
+
+2. To process the received search command on the server side, add an event handler to the `SearchCommandReceived` event of the [Tizen.Multimedia.Remoting.MediaControlServer](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlServer.html) class:
+
+    ```csharp
+    MediaControlServer.SearchCommandReceived += (s, e) =>
+    {
+        foreach (var condition in e.Command.Conditions)
+        {
+            Log.Info("MC", $"{condition.Category}, {condition.ContentType}, {condition.Keyword}");
+        }
+
+        MediaControlServer.Response(e.Command, (int)ErrorCode.None);
+    };
+    ```
+
+To send a custom command from the server and to process it on the client side, follow these steps:
+1.  To send a search command on the server side, use the `RequestAsync()` method of the [Tizen.Multimedia.Remoting.MediaControlServer](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlServer.html) class:
+
+    ```csharp
+    MediaControlServer.RequestAsync(new CustomCommand("CustomAction"));
+    ```
+
+2. To process the received custom command on the client side, add an event handler to the `CustomCommandReceived` event of the [Tizen.Multimedia.Remoting.MediaController](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaController.html) class:
+
+    ```csharp
+    controller.CustomCommandReceived += (s, e) =>
```

Comment:
Just a question.

Is this "controller" OK in this sample code?

controller.CustomCommandReceived

Reply:
I think it's not problem since it's described like following code.

```
var controller = mediaControllerManager.GetActiveControllers().First();
```

Reply:
Sorry, @hsgwon. I can't see your code.
Instead, I see below in Prerequisites section.
```
var mediaController = mediaControllerManager.GetActiveControllers().First();
```

It is mediaController, not controller.

Would you please re-check?

Reply:
I'm sorry, I checked it in different branch(tizen_5.5_prepare). 
I'll fix it.